source/documentation/deploying_apps/deploying_django.md

```diff
@@ -2,7 +2,7 @@
 
 This section explains how to deploy an app using the Django framework. You may also need to refer to the [Cloud Foundry documentation about the Python buildpack](https://docs.cloudfoundry.org/buildpacks/python/index.html) [external link].
 
-Note that the only database service currently supported by PaaS is PostgreSQL. If your Django app requires a database, it must be able to work with PostgreSQL.
+Note that the PaaS currently supports the PostgreSQL, MySQL and MongoDB database services. If your Django app requires a database, it must be able to work with one of these.
```

Comment:
Should we link to [#deploy-a-backing-or-routing-service](https://docs.cloud.service.gov.uk/#deploy-a-backing-or-routing-service) as an authoritative list of services rather than duplicating the list in each of these sections? Otherwise it's going to be out-of-date again in the future.